charts/snowplow-iglu-server/templates/iglu-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.global.deployTraefikIngress }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "iglu.app.name" . }}
+  annotations:
+    traefik.ingress.kubernetes.io/router.entrypoints: {{ .Values.service.azure.traefikEntrypoint }}
+
+spec:
+  rules:
+    - host: {{ .Values.service.azure.hostname }}
```

Comment:
Has `azure` in the path

Reply:
Apologies , I wasn't ran a test for this change. Updated path , also did a test and confirmed that all paths are correct.